python/requirements.txt

```diff
@@ -1,2 +1,3 @@
 # This file should list any python package dependencies.
-requests
\ No newline at end of file
+requests
+future
```

Comment:
As I mentioned in the above description, Python3 doesn't have `raw_input`. In order to be compatible code between Python2 and Python3, I think it would be better to use `python-future`.

http://python-future.org/compatible_idioms.html